docs/topics/db/transactions.txt

```diff
@@ -232,6 +238,17 @@ Django provides a single API to control database transactions.
     is especially important if you're using :func:`atomic` in long-running
     processes, outside of Django's request / response cycle.
 
+.. warning:: Durability and testing
+
+    :class:`django.test.TestCase` disables the durability check to allow
+    testing durable atomic blocks in a transaction for performance reasons.
+    To test the durability guarantees, use
+    :class:`django.test.TransactionTestCase` instead.
```

Comment:
This might want covering somewhere in the documentation for `TestCase` as well?

Reply:
Can do.

Reply:
The [`TestCase` documentation](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#testcase) already has:

> Wraps the tests within two nested atomic() blocks: one for the whole class and one for each test. Therefore, if you want to test some specific database transaction behavior, use TransactionTestCase.

I'll add a `versionchanged` section too though.